### PR TITLE
 build: Fix build after rust-lang/rust#60951 

### PR DESCRIPTION
**Pull request by lzutao**

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -1,3 +1,4 @@
+#![deny(rust_2018_idioms)]
```
Comment (RalfJung) [inline src/lib.rs]: Please make this a `warn`, I don't think I want this to fail when I am working locally. if you want you can make CI use `-D warnings`.

Also please move this down below the feature gate, to where there already is an `allow`.

**Comment by RalfJung** [comment]
Other than this nit, looks good. Thanks!

File: src/lib.rs (hunk 1 above)
Comment (RalfJung) [reply src/lib.rs]: Oh, and if you intend to deny warnings on CI, please make that a separate PR.

**Comment by lzutao** [comment]
I removed the second commit (to force to use 2018 edition idioms).

**Comment by RalfJung** [comment]
That also works. I am still in favor of *using* 2018 idioms, I just don't like warnings to stop things from compiling when I am developing. So I'd be happy if you could re-submit that commit, with my comments applied, as a new PR. :)

**Comment by RalfJung** [comment]
Timeout on macOS.

@bors retry

**Pull request merged**